data-and-functions-1/src/functions/getUserById.js

```diff
@@ -0,0 +1,9 @@
+const getUserById = (data, id) => {
+  if (data == null || id == null || data.users == null) {
+    return null
+  }else{
+    return data.users.find((users) => users.id === id)
```

Comment:
Change the `users` parameter in your `find` to `user` or `u`, since it's just a single user object:
```
return data.users.find(u => u.id === id)
```